### DISCUSSION TITLE
Eclipse finally have arm64 *Early Access* builds

### DISCUSSION THREAD

**Discussion by zigang93** [dbeaver/dbeaver#12103] [General]
https://bugs.eclipse.org/bugs/show_bug.cgi?id=565690
I saw eclipse already release the arm64.. 
which mean dbeaver finally able to start support mac m1 ?

**Comment by tati-kru** [comment]
This means that DBeaver supports arm64 as well. You can download DBeaver CE for ARM on dbeaver.io. Unfortunately, version for mac m1 hasn't been implemented yet. But we are looking forward to hearing about it in the near future.
